### PR TITLE
Improve view of data

### PR DESCRIPTION
**Pull request by stevenernst131**
image

image

### CODE REVIEW

File: AngularApp/src/app/modules/dashboard/signal-container/signal-container.component.ts [hunk 1]
```diff
@@ -59,4 +61,37 @@ export class SignalContainerComponent implements OnInit {
     this._router.navigate(['edit'], navigationExtras);
   }
 
+  //TODO: REMOVE
```
Comment (shekhar-gupta-dev) [inline AngularApp/src/app/modules/dashboard/signal-container/signal-container.component.ts]: did you forgot to remove?

File: AngularApp/src/app/shared/services/diagnostic-api.service.ts [hunk 2]
```diff
@@ -23,13 +23,13 @@ export class DiagnosticApiService {
 
   public getDetector(resourceId: string, detector: string, resourceSpecificQueryString: string): Observable<DetectorResponse> {
     let timeParameters = this._getTimeQueryParameters();
-    let path = `v4/${resourceId}/diagnostics/detectors/${detector}?${resourceSpecificQueryString}${timeParameters}`;
-    return this.invoke<DetectorResponse>(path, HttpMethod.GET);
```
Comment (shekhar-gupta-dev) [inline AngularApp/src/app/shared/services/diagnostic-api.service.ts]: need to revert this?

**Comment by ShekharGupta1988** [comment]
@stevenernst131  - yo, checkout my email. Looks like one more fix needs to be done to fix the side menu for detectors.

**Comment by stevenernst131** [comment]
Yes I will make the fixes above, forgot to remove those. I will try to deploy a working update of applens for puneet.

**Comment by stevenernst131** [comment]
Hey @ShekharGupta1988, I have made the fixes for the problems that were left over from my differing API endpoints. This is the version I published last night, with the temporary code removed.

**Pull request merged**